README.md

```diff
@@ -68,36 +68,86 @@ The frontend will display the contents of the results plugin for the AutoGov sta
 
 ### Installation into Backstage
 
+#### Update Configuration
+
+1. Add an autogov config section to your `app-config.yaml`
```

Comment:
Mentioning/linking to the bellow [Configuration Options](#ConfigurationOptions) would be a good idea.

Reply:
I'm not seeing anything "below"? (just seeing a link for this PR)

https://github.com/liatrio/backstage-plugin-autogov/pull/42#ConfigurationOptions

Reply:
maybe link to this? https://backstage.io/docs/conf/

Reply:
https://github.com/liatrio/backstage-plugin-autogov/blob/e05f89dd6fdf3aea45aef4563c5d4c9c581bbc7f/README.md?plain=1#L318

Reply:
Basically, by pulling to configuration block up to the top, it would be good to link to the bottom where it was on what the value options are.

Reply:
Oh gotcha! (totally on the same page now)